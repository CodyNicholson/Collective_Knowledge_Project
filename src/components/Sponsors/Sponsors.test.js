//This file was begun using code from Link.react.test.js, from the Jest Testing React Apps Guide
//https://facebook.github.io/jest/docs/en/tutorial-react.html
import React from 'react';
import SponsorComponent from './SponsorComponent';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

test('Tags module test', () => {
  const component = renderer.create(
	<SponsorComponent children="thing1;thing2;thing3;thing4u">
	</SponsorComponent>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});