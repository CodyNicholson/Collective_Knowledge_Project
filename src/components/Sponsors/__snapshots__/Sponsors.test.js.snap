// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tags module test 1`] = `
<div
  className="commentContainer"
>
  <div
    className="projFlexList"
  >
    <div
      className="textContent"
    >
      <h2
        className="title"
      >
        thing1
      </h2>
      <div
        className="description"
      >
        thing2
      </div>
      <button
        className="button-primary"
        onClick={[Function]}
      >
        Edit
      </button>
      <button
        className="button-danger"
        onClick={[Function]}
      >
        Remove
      </button>
    </div>
  </div>
</div>
`;
